drivers/sensor/bh1745/bh1745_trigger.c

```diff
@@ -0,0 +1,198 @@
+/*
+ * Copyright (c) 2019 Nordic Semiconductor ASA
+ * Copyright (c) 2022 Grzegorz Blach
+ *
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+#include <device.h>
+#include <drivers/gpio.h>
+#include <drivers/i2c.h>
+#include <sys/util.h>
+#include <kernel.h>
+#include <drivers/sensor.h>
+#include "bh1745.h"
+
+#include <logging/log.h>
+LOG_MODULE_REGISTER(BH1745_TRIGGER, CONFIG_SENSOR_LOG_LEVEL);
+
+#define DT_DRV_COMPAT rohm_bh1745
+
+/* Callback for active sense pin from BH1745 */
+static void bh1745_gpio_callback(const struct device *dev,
+				 struct gpio_callback *cb,
+				 uint32_t pins)
+{
+	struct bh1745_data *drv_data =
+		CONTAINER_OF(cb, struct bh1745_data, gpio_cb);
+
+	gpio_pin_interrupt_configure(dev, DT_INST_GPIO_PIN(0, int_gpios),
```

Comment:
Please use the dt variant for all gpio transactions (`gpio_pin_interrupt_configure_dt` and friends)

Reply:
Done